test/reporter-urlparams-hidepassed.js

```diff
@@ -0,0 +1,30 @@
+if ( !location.search ) {
+	location.replace( "?hidepassed=true" );
+}
+
+QUnit.module( "urlParams hidepassed module", function() {
+	QUnit.test( "passed", function( assert ) {
+		assert.ok( true );
+	} );
+	QUnit.test( "passed", function( assert ) {
+		assert.ok( true );
+	} );
+	QUnit.test( "passed", function( assert ) {
+		assert.ok( true );
+	} );
+	QUnit.test( "passed", function( assert ) {
+		assert.ok( true );
+	} );
+	QUnit.test( "passed", function( assert ) {
+		assert.ok( true );
+	} );
+	QUnit.test( "passed", function( assert ) {
+		assert.ok( true );
+	} );
+
+	QUnit.test( "passed", function( assert ) {
+
+		// we have to set it to 1 becuase there is currently one item being rendered, this one as it is in progress
```

Comment:
There's only one item being rendered because all the test name are the same. Is this intentional?

Reply:
Well there should be 7 divs realistically, only one should exist as it is the current one.

Reply:
Fixed in https://github.com/qunitjs/qunit/commit/f64b5a113bac52eb1bf3dfbf48f896fe2da6bfd4.